tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3779,6 +3883,34 @@ <h2>Show similarity to selected datapoint</h2>
           val / this.getTotalEntriesInConfCounts(confCounts) * 100);
       },
 
+      /**
+       * Returns the F1 score for a given confusion matrix
+       * from the threshold selected, model index, and facet to view.
+       */
+       getF1ModelIndex: function(inferenceStats, thresholds, modelInd, item) {
+        // TODO(jameswex): This unnecessarily recalculates confusion matrix.
+        // Can speed this up.
+        const formatter = d3.format(",.2f");
+        const confCounts = this.getConfusionCountsModelIndex(
+          inferenceStats, thresholds, modelInd, item);
+        if (Object.keys(confCounts).length == 0) {
+          return 0;
+        }
+        if (confCounts['Yes']['Yes'] == 0) {
```

Comment:
might be more readable (or more maintainable) to assign this to a variable.
```js
const truePositive = confCoujnts['Yes']['Yes'];
...
```

Reply:
done